daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-remote-dev-containers.md

```diff
@@ -0,0 +1,31 @@
+---
+type: docs
+title: "Visual Studio Code remote dev containers"
+linkTitle: "Remote dev containers"
+weight: 20000
+description:  "How to setup a remote dev container environment with Dapr"
+---
+
+The Visual Studio Code [Remote Containers extension](https://code.visualstudio.com/docs/remote/containers) lets you use a Docker container as a full-featured development environment without installing any additional frameworks or packages to your local filesystem.
+
+Dapr has pre-built Docker remote containers for each of the language SDKs. You can pick the one of your choice for a ready made environment. Note these pre-built containers automatically update to the latest Dapr release.
```

Comment:
Is this true? I thought this was only Node and C#? Is there one for Java, Node and Python? Please confirm.

Reply:
As a follow up note to this, can we check that these are still valid and pull in the latest Dapr SDK when used? Check with Philip who generate these.

Reply:
Yeah you're right only Node and C#, I had just trusted the details that were in the current docs. And from my understanding of the code I believe so when I followed the procedures the Docker file didn't seem dapr version specific but i will ask Phillip since I should've confirmed!